utils/export_onnx_model.py

```diff
@@ -0,0 +1,49 @@
+from torch.autograd import Variable

```

Comment:
I believe we need to add the Copyright header here. Can you copy from `utils/download_test_db.py`?

Reply:
Done